Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -409,6 +409,16 @@ internal static void Update()
             if (!active)
                 return;
 
+            // Check for dead clients but exclude the host client because it
+            // doesn't ping itself and therefore may appear inactive.
+            foreach (NetworkConnectionToClient conn in connections.Values)
+            {
+                if (!(conn is ULocalConnectionToClient))
```

Comment:
We don't need to check for this here, we can just override `CheckForActivity` inside ULocalConnectionToClient

Reply:
Funny enough, I had an override in that at first, full of comments on why it was there, why it did nothing, and why it couldn't be removed, then I realized I could check for it here, and took it out.

Reply:
it's a virtual bool now.